model/core/prelude_mem.sail

```diff
@@ -17,8 +17,6 @@ $include <concurrency_interface.sail>
 
 enum write_kind = {
   Write_plain,
-  Write_RISCV_release,
-  Write_RISCV_strong_release,
```

Comment:
This might break the memory model in ISLA.
https://github.com/rems-project/isla/blob/dcece5f1862b961a8b0f205db9d6a0b461bc4335/configs/riscv32.toml#L57
https://github.com/rems-project/isla/blob/dcece5f1862b961a8b0f205db9d6a0b461bc4335/configs/riscv64_ubuntu.toml#L57